lib/administrate/field/belongs_to.rb

```diff
@@ -8,7 +8,7 @@ def self.permitted_attribute(attr)
       end
 
       def permitted_attribute
-        self.class.permitted_attribute(attribute)
+        foreign_key || self.class.permitted_attribute(attribute)
```

Comment:
@jcmorrow i ended up with 
```ruby 
def foreign_key
  options.fetch(:foreign_key, :"#{attribute}_id")
end
```
on the instance level of BelongsTo, i found dangerous to rewrite `permitted_attributes` because it's used by BaseDashobard and Deffered classes.